### ISSUE TITLE
When without use cdn,mermaid can't work

### ISSUE THREAD

**Issue by ericma15** [hackmdio/hackmd#600]
When use cdn , mermaid can work,but when without it,it can't work.It said "Cannot read property 'parse' of undefined". Finally I found that the mermaid.min.js I get by npm is not the same as that I got from cdn.It is newer. When I replace the file by the one I got from cdn.  It worked.The function "prase" has been deleted.The new version of mermaid may deprecate some functions now use. So please change these code and fix it. Thx.

**Comment by SISheogorath** [comment]
Can you share an example snippet where your error message can be reproduced? I use the current master and was unable to reproduce the issue using the `/feature` page.

**Comment by SISheogorath** [comment]
Confirmed.

Issue was introduced by https://github.com/knsv/mermaid/commit/a5b71455271aef1180eefa1054d751dea1885e3f#diff-67ae90c5144c55348a3cbdb078240454

We need to use the mermaidAPI inside the mermaid scope.

**Comment by SISheogorath** [comment]
We need to wait for cdnjs to provide the updated version. Should happen soon.

@PeterDaveHello will hopefully take care of this part ^^

**Comment by PeterDaveHello** [comment]
See #607 :)

**Comment by SISheogorath** [comment]
Took a moment but is completely fixed for now :smile: 

Thanks a lot for reporting!

**Comment by ericma15** [comment]
Thanks. But in fact, this problem still don't be solved.
```
index-pack.js:13279 TypeError: Cannot read property 'parse' of undefined
    at HTMLDivElement.<anonymous> (index-pack.js:13269)
    at Function.each (common.js:675)
    at jQuery.fn.init.each (common.js:470)
    at finishView (index-pack.js:13259)
    at updateViewInner (index-pack.js:100791)
    at Socket.<anonymous> (index-pack.js:99972)
    at Socket.Emitter.emit (index-pack.js:69366)
    at Socket.onevent (index-pack.js:107614)
    at Socket.onpacket (index-pack.js:107572)
    at Manager.<anonymous> (index-pack.js:106303
```
the mermaid is 7.1.0 now

**Comment by SISheogorath** [comment]
did you use the latest commit b3c66d4a100899d72a201ad9e95a137c2ceeccf4?

**Comment by SISheogorath** [comment]
Because this exact issue should be solved but introduced #610 😅

**Comment by ericma15** [comment]
Yes, I see. I know now. Thx.
